test/contrib/bigquery_test.py

```diff
@@ -38,8 +38,7 @@
 
 @attr('gcloud')
 class TestLoadTask(bigquery.BigqueryLoadTask):
-    client = None
-
+    client = MagicMock()
```

Comment:
Is it possible using a mock here broke the existence checks on tables in the integration suite?

See here:
https://travis-ci.org/spotify/luigi/jobs/92593476#L208